### DISCUSSION TITLE
웹사이트 테스트

### DISCUSSION THREAD

**Discussion by kwnjisu** [community/community#135991] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

1일차 초보입니다. 웹사이트를 테스트 목적으로 크롬 개발자모드로 요소를 복사 해서 테스트 하고 싶습니다. 하지만 특정 웹사이트는 전체 화면이 잘 뜨고 특정 웹사이트는 css?가 적용이 되지 않는것 같습니다. 웹사이트 html을 복사로만 테스트는 어렵나요?

**Comment by LiteBrite82** [comment]
Hi @kwnjisu , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
